LEGO1/mxbitmap.cpp

```diff
@@ -0,0 +1,13 @@
+#include "mxbitmap.h"
+
+// OFFSET: LEGO1 0x100bc980
+MxBitmap::MxBitmap()
+{
+    // fixme: vtable, mxcore??
```

Comment:
This class derives from MxCore, that's why you're seeing that

Reply:
So what does that mean, like, is there an "extends" of C++?

Reply:
It means class declaration is (to be) `MxBitmap : public MxCore` (`public` could also be `protected` or `private`).

That results in
1. The subclass inheriting all fields of the baseclass.
2. The compiler generating code to invoke MxCore's ctor/dtor from the subclass's ctor/dtor, before/after setting the vfptr to the table for the actual type.

Note that the baseclass' ctor/dtor may or may not be inlined, to the point even setting the vfptr could be elided. In the case of MxCore's ctor and dtor, though, they are never inlined in the dll, so MxBitmap's ctor 1. calls MxCore's ctor, 2. sets the vfptr to MxBitmap vtable. See follow up comment for the header file below.